Combobox.js

```diff
@@ -395,11 +402,23 @@ define([
 			}
 			if ("value" in oldValues) {
 				if (!this._justValidated) {
-					this._validateInput(false, true);
-				} else {
-					delete this._justValidated;
+					if (this.list) {
+						// INFO: if list is already created and attached, then we can validate the `value` value
+						this._validateInput(false);
+					} else {
+						// INFO: otherwise we have to delay its evaluation.
+						this._processValueAfterListInit = true;
+					}
 				}
 			}
+			if ("_justValidated" in oldValues) {
+				if (this._justValidated) {
+					this._justValidated = false;
+				}
+			}
```

Comment:
This is confusing to me, why don't you just do:

```
this._justValidated = false;
```

all the time, without the `if()` statements?